packages/astro/src/core/build/buildPipeline.ts

```diff
@@ -61,7 +61,8 @@ export class BuildPipeline extends Pipeline {
 				routeCache: staticBuildOptions.routeCache,
 				site: manifest.site,
 				ssr,
-				streaming: true,
+				// NOTE: there's no need to enable streaming in SSG
+				streaming: !ssr,
```

Comment:
Isn’t this backwards? Should be:
```suggestion
				streaming: ssr,
```